### PR TITLE
refactor(client): Alphabetize deps and hash keys in metrics.js

### PR DESCRIPTION
**Pull request by shane-tomlinson**

### CODE REVIEW

**Comment by shane-tomlinson** [comment]
@philbooth - could I get a review of this?

File: app/scripts/lib/metrics.js [hunk 1]
```diff
@@ -123,8 +123,9 @@ define([
     this._env = options.environment || new Environment(this._window);
 
     this._lastAbLength = 0;
-    // if navigationTiming is supported,
-    // the baseTime will be from navitgationTiming.navigationStart, otherwise Date.now().
+
+    // if navigationTiming is supported, the baseTime will be from
+    // navitgationTiming.navigationStart, otherwise Date.now().
```
Comment (philbooth) [inline app/scripts/lib/metrics.js]: Pedantry alert: Typo here.

Comment (shane-tomlinson) [inline app/scripts/lib/metrics.js]: More of this!

**Comment by philbooth** [comment]
Looks ruddy marvellous to me. :+1:

**Comment by shane-tomlinson** [comment]
Thanks @philbooth - I'll make sure all the CI tests pass, then merge.

**Pull request merged**